Packs/1_1_3SuperchargingXSOARWithAnsible/Integrations/MicrosoftGraphMailXmocky/MicrosoftGraphMailXmocky.yml

```
@@ -0,0 +1,762 @@
+      - Normal
+      - High
+    - description: 'A comma-separated list of additional headers in the format: headerName:headerValue.
+        For example: "headerName1:headerValue1,headerName2:headerValue2".'
+      isArray: true
+      name: headers
+    - description: A comma-separated list of War Room entry IDs that contain files,
+        which are used to attach files for the email to send. For example, attachIDs=15@8,19@8.
+      isArray: true
+      name: attachIDs
+    - description: A comma-separated list of names of attachments to display in the
+        email to send. Must be the same number of elements as attachIDs.
+      isArray: true
+      name: attachNames
+    - description: A comma-separated list of CIDs to embed attachments within the
+        actual email.
+      isArray: true
+      name: attachCIDs
+    - description: The email address from which to send the email.
+      name: from
+    description: Sends an email using Microsoft Graph.
+    name: send-mail
+    outputs:
+    - contextPath: MicrosoftGraph.Email.internetMessageHeaders
+      description: The email headers.
+      type: String
+    - contextPath: MicrosoftGraph.Email.body
+      description: The body of the email.
+      type: String
+    - contextPath: MicrosoftGraph.Email.bodyPreview
+      description: The body preview of the email.
+      type: String
+    - contextPath: MicrosoftGraph.Email.subject
+      description: The subject of the email.
+      type: String
+    - contextPath: MicrosoftGraph.Email.flag
+      description: The flag status of the email.
+      type: String
+    - contextPath: MicrosoftGraph.Email.importance
+      description: The importance status of the email.
+      type: String
+    - contextPath: MicrosoftGraph.Email.toRecipients
+      description: The 'to' recipients of the email.
+      type: String
+    - contextPath: MicrosoftGraph.Email.ccRecipients
+      description: The CC recipients of the email.
+      type: String
+    - contextPath: MicrosoftGraph.Email.bccRecipients
+      description: The BCC recipients of the email.
+      type: String
+  - arguments:
+    - description: The ID of the message.
+      name: ID
+      required: true
+    - description: The comment of the replied message.
+      name: body
+      required: true
+    - description: A comma-separated list of email addresses for the 'to' field.
+      isArray: true
+      name: to
+      required: true
+    - description: The email address from which to reply.
+      name: from
+      required: true
+    description: The replies to the recipients of a message.
+    name: msgraph-mail-reply-to
+  - arguments:
+    - description: The ID of the draft email.
+      name: draft_id
+      required: true
+    - description: The email address from which to send the draft.
+      name: from
+      required: true
+    description: Sends a draft email using Microsoft Graph.
+    name: msgraph-mail-send-draft
+  dockerimage: demisto/crypto:1.0.0.303
```

```suggestion
  dockerimage: demisto/crypto:1.0.0.13199
```
The latest docker image tag in docker hub is: 1.0.0.13199 , If this version works for you please update the docker image.

---

I actually didn't intend to submit the MicrosoftGraphMailXmock integration. This is something that was in the XSOAR instance provided for the hackathon. I will remove this integration from the my PR.